File: templates/python/es/text/code/cornerBeepers.py
# Archivo: Esquina Conos .py
# -----------------------------
# cono un cono en cada esquina
from karel.stanfordkarel import *
def main():
   # repite el cuerpo 4 veces
   for i in range(4):
	   poner_cono()
	   moverse()
	   moverse()
	   moverse()
	   girar_izquierda()

File: templates/java/es/text/code/cornerBeepers.java
/*
 * Archivo: EsquinaConos.java
 * -------------------------
 * cono un cono en cada esquina.
 */
import stanford.karel.*;
public class EsquinaConos extends Karel {
   public void run() {
      /* repite el cuerpo 4 veces */
      for(int i = 0; i < 4; i++) {
      	ponerCono();
      	moverse();
      	moverse();
      	moverse();
      	girarIzquierda();
      }
   } 
}